Domain Connect Spec Draft.adoc

```diff
@@ -1952,6 +1952,50 @@ record of "xyz.bar" (or "xyz.bar.example.com." in bind format).
 A value of @ for host in the template is a placeholder for an empty value. In other words @
 would point to "bar.example.com." when the same template is applied to domain=example.com and host=bar.
 
+In summary.
+
+....
+       +------+
+       | user |
+       +------+
+          |
+          v
+   +----------------------------+
+   | https://.../apply?host=xyz |
+   +----------------------------+
+          |
+          v
+   +----------------------------------------+
+   | get domain from template: example.com. |
+   +----------------------------------------+
+          |
+          v
+   +------------------------------+
+   | does template include: host? |
+   +------------------------------+
+          |                    |
+         no                   yes
+          v                    |
+   +-------------------------+ |
+   | apply: xyz.example.com. | |
+   +-------------------------+ |
+            ^                  v
+            |   +----------------+
+           yes--| is the host: @ |
+                +----------------+
+                  | no
+                  v
+   +-----------------------------+
+   | get host from template: bar |

```

Comment:
Somehow would be good to be more indicative which part is an algorithm and which one is an example